pkgs/dartpad_shared/lib/backend_client.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:http/http.dart';
+
+import 'headers.dart';
+
+class DartServicesClient {
+  final Client _client = Client();
+  static Map<String, String> _headers =
+      DartPadRequestHeaders(enableLogging: true).encoded;
+
+  /// Turns off backend logging for all future requests.
+  static void turnOffBackendLogging() {
+    _headers = DartPadRequestHeaders(enableLogging: false).encoded;
+  }
+
+  void dispose() => _client.close();
```

Comment:
nit: Keep `close()` for consistency - we match the HttpClient API for other methods on this class.